cmd/flux/bootstrap_github.go

```diff
@@ -111,7 +111,11 @@ func init() {
 func bootstrapGitHubCmdRun(cmd *cobra.Command, args []string) error {
 	ghToken := os.Getenv(ghTokenEnvVar)
 	if ghToken == "" {
-		return fmt.Errorf("%s environment variable not found", ghTokenEnvVar)
+		var err error
+		ghToken, err = readPasswordFromStdin("Please type your GitHub personal access token: ")
```

Comment:
"[T]ype" sounds strange here. Perhaps "enter" instead? It's unlikely that anyone will be typing the token value.

Also, consider mentioning the common acronym "PAT."
```suggestion
		ghToken, err = readPasswordFromStdin("Please enter your GitHub personal access token (PAT): ")
```